developing/direnv/README.md

```diff
@@ -0,0 +1,131 @@
+# [Tools and Practice](../README.md) / direnv
+
+## Overview
+
+[direnv](https://direnv.net/) is an MIT-licensed command-line tool that addresses the hassle of setting per-project environment variables.
+
+## Installation
+
+Via homebrew:
+
+     brew install direnv
+
+## Usage
+
+Create an `.envrc` file in a directory with some environment variables your project needs:
```

Comment:
might be worth adding something re: putting this in the global gitignore

if we already have that explained under source control, then you can point back to there? :)

Reply:
Hmmm. Sometimes the `.envrc` has things that should be source controlled though. On MilMove they did something pretty neat to resolve that tension so I can put something in about it.

Reply:
@sojeri does 48ef255 help?

Reply:
sure, that addresses the issue enough for me :)